primitives/npos-elections/src/tests.rs

```diff
@@ -792,6 +792,13 @@ mod assignment_convert_normalize {
 
 mod score {
 	use super::*;
+	use crate::ElectionScore;
+	use sp_arithmetic::PerThing;
+
+	fn is_score_better(this: [u128; 3], that: [u128; 3], p: impl PerThing) -> bool {
```

Comment:
Could switch to `ElectionSocre`, or (u128, u128, u128) for shorter code.

Reply:
same with https://github.com/paritytech/substrate/pull/10834#discussion_r805736070